### DISCUSSION TITLE
Getting 10s timeout on kubectl command that takes 11s

### DISCUSSION THREAD

**Discussion by troysas** [Azure/Bridge-To-Kubernetes#125] [Q&A] [answered]
When connecting to the cluster using the Kubernetes status button at the bottom of the VSCode UI, I get this error:

Error: Failed to validate the requirements to replicate resources locally: An unexpected error occurred: ''kubectl' timeout after 10000ms when running exec my-client-679d885556-znxr9 -c my-client-container -n blahk8stroy -- env'

When I run this command via terminal, it consistently takes an average of 11s:
real	0m10.934s
user	0m0.077s
sys	0m0.057s

Is there some way to change the timeout? or some other way to work around this.  I thought it might be because my VPN for work isn't split tunneling but even without VPN connected I still get this time.

Thanks.

**Comment by hsubramanianaks** [comment, marked as answer]
@troysas can you raise an issue with details if you are still facing this issue ? Sorry just seeing this. Thanks.
